Reference-Platform/Platforms/Enterprise/ReleaseNotes-17.12.md

```diff
@@ -59,8 +59,6 @@ The goal of the Linaro Enterprise Reference Platform is to provide a fully teste
 - [3499](https://bugs.linaro.org/show_bug.cgi?id=3499) [HP-m400] CPU\*: failed to boot: -6
 
 ### DPDK is released with the following known bugs:
-- [2657](https://bugs.linaro.org/show_bug.cgi?id=2657) [D03] [D05] Confusing Ethernet port sequence
-- [3169](https://bugs.linaro.org/show_bug.cgi?id=3169) DPDK: can't enable sr-iov on D05 with Intel 82599
```

Comment:
Please leave this bug in here for Xinliang to deal with.

Reply:
Ok lets close this one and use #59 instead.
https://github.com/Linaro/documentation/pull/59